sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseNewFragment.java

```diff
@@ -249,6 +251,10 @@ public void onLayoutBinding(FragmentCaseNewLayoutBinding contentBinding) {
 			if (e.getValue() == TypeOfPlace.FACILITY) {
 				contentBinding.facilityTypeGroup.setValue(FacilityTypeGroup.MEDICAL_FACILITY);
 				contentBinding.caseDataFacilityType.setValue(FacilityType.HOSPITAL);
+				User user = ConfigProvider.getUser();
+				if (!user.hasUserRole(UserRole.HOSPITAL_INFORMANT)) {
```

Comment:
Instead of checking the role, please check the user's jurisdiction level (which should be facility); that's more future-proof since we'll get rid of default user roles at some point

Reply:
done